package.json

```diff
@@ -23,50 +23,50 @@
     "broccoli-concat": "0.0.13",
     "broccoli-funnel": "^1.0.0",
     "broccoli-merge-trees": "^1.0.0",
-    "broccoli-static-compiler": "0.2.1",
+    "broccoli-static-compiler": "0.2.2",
```

Comment:
static-compiler is deprecated in-favor of broccoli-funnel (maybe a future PR should address?